app/workers/mail_worker.rb

```diff
@@ -0,0 +1,7 @@
+class MailWorker
+  include Sidekiq::Worker
+
+  def perform
+    RentMailer.rent_created.deliver_now
```

Comment:
Instead of creating a whole worker for this, you can just do `RentMailer.rent_created(rent.id).deliver_later`, activejob/activemailer has some workers for this, and it's automagical! It's basically the same implementation you did, but easier and without extra-classes